Source/DataSources/KmlDataSource.js

```diff
@@ -595,6 +595,7 @@ import WallGraphics from './WallGraphics.js';
 
         var resource;
         if (defined(uriResolver)) {
+            href = href.replace(/\\/g, '/');
```

Comment:
I think it's worth adding a short inline comment explaining why backslashes need to be converted here.